docs/docs/contribution-juno.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title : Contribution to Juno
+---
+
+# Contributions Guidelines
+
+See [Engineering Guidelines](https://www.notion.so/Engineering-Guidelines-f916deb46ee242a0afaf36cc436e4c73) for more details
+
+## 1. Create a branch off of `main`
```

Comment:
I don't understand well what you are trying to guide here. I suggest to put here a header for something like 

"How to contribute" or "Adding new issues"
1. File an issue to describe the feature you want to add, the bug you want to fix, the code refactor you want to do, etc and with this start a discussion. Be sure that this is something the core team would want to have it done before start working on it.
2. Create a branch ....
3. Writing code ...
4. Create a PR ...
5. Submit for review ...